src/templates/base/index.html

```diff
@@ -1,8 +1,8 @@
 {% extends "%s/%s/base.html" % (lang, year) %}
 
-{% block image_url %}https://almanac.httparchive.org/static/images/home-hero-bg.png{% endblock %}
-{% block image_height %}562{% endblock %}
-{% block image_width %}715{% endblock %}
+{% block image_url %}https://almanac.httparchive.org{{ get_versioned_filename('/static/images/home-hero-' + year + '.png') }}{% endblock %}
```

Comment:
@bazzadp I'm not sure if I should be using `get_versioned_filename` here. I don't see pngs in `last_updated.json`, but should there be?

Reply:
Personally I don't think it's worth tracking images. Very, very, very rare there's breaking changes in these unlike CSS or JS.

Reply:
I was asking for its cache-busting side-effect. Images can be long-lived and it would be nice to ensure that any changes take effect immediately after a release.

Reply:
That's not a side-effect - that's the whole reason we have that 😁

Yes we could do this, but also then should do for all the figures across all the chapters and years. And that makes the file rather large and its read at server start up so could impact the server - especially as it grows and grows over the years.

We have a maximum cache tie of 3 hours (I'm against really long lived cache times to be honest precisely for this reason) so not sure it's worth the effort.

Reply:
Would you suggest hard-coding the URL in that case?

Also one other question, WDYT about saving the year-specific images under their respective static/image/[year] subdirectories?

Reply:
> Would you suggest hard-coding the URL in that case?

Can;t be fully hardcoded as you now have a different one for 2019 and 2020 but could be semi-hardcoded with 

```suggestion
{% block image_url %}https://almanac.httparchive.org/static/images/home-hero-{{ year }}.png{% endblock %}
```

Probably more efficient to do that than your current function.

> Also one other question, WDYT about saving the year-specific images under their respective static/image/[year] subdirectories?

Yeah we should probably do that. I've an outstanding question/TODO in #1558 to clean up chapter images and, depending how we do that, could sort this out at same time. For this PR, I say keep it as is for now.

Reply:
Shall I commit this suggestion and then merge this @rviscomi ?